src/clang-cpp-frontend/clang_cpp_adjust.h

```diff
@@ -5,15 +5,24 @@
 
 /**
  * clang C++ adjuster class for:
+ *  - symbol adjustment, dealing with ESBMC-IR `symbolt`
  *  - expression adjustment, dealing with ESBMC-IR `exprt` or other IRs derived from exprt
  *  - code adjustment, dealing with ESBMC-IR `codet` or other IRs derived from codet
+ *  - implicit code generation, e.g. generate implicit GOTO code for vptr initializations in ctors
+ *
+ * Some ancillary methods to support the expr/code adjustments above
  */
 class clang_cpp_adjust : public clang_c_adjust
 {
 public:
   explicit clang_cpp_adjust(contextt &_context);
   virtual ~clang_cpp_adjust() = default;
 
+  /**
+   * methods for symbol adjustment
+   */
+  void adjust_symbol(symbolt &symbol);
```

Comment:
This comment is just repeating what's in the method's name. That is quite useless. Also it is just a single method.

Reply:
It may look useless at first glance in this code snippet, however it's actually indicating "the functions below are used for symbol adjustment" and to be consistent with the structure of `src/clang-c-frontend/clang_c_adjust.h` (i.e. The `adjust_*` functions are grouped together in clang_cpp_adjust class. ). 
It would make a lot sense in `src/clang-cpp-frontend/clang_cpp_adjust.h`.

Reply:
Well, if "the functions below are used for symbol adjustment" is meant to be indicated, maybe it should just say that. :)